src/core_tests.rs

```diff
@@ -264,18 +264,17 @@ fn poll_all(nodes: &mut [TestNode], clients: &mut [TestClient]) -> bool {
     loop {
         let mut n = false;
```

Comment:
Since we're in here anyway, could `n` be changed to something more meaningful, e.g. `events_occurred` or something?